Cleanups before internal testing (#629)
0xPolygonMiden/miden-base: docs/introduction/get-started/create-account-use-faucet.md

!!! tip
      Check the [Miden client documentation](https://docs.polygon.technology/miden/miden-client/cli-reference/) for more information.

1. Open your terminal and create a new directory to store the Miden client.

    ```sh
    mkdir miden-client-2
    cd miden-client-2
    ```

2. Build and install the client using cargo:

      ```shell
      cargo install miden-client --features testing,concurrent
      ```

   You can now use the `miden-client` command.

> 3. Initialize the client and point it to the Miden testnet. IP: `18.203.155.106`

Reviewer: ```suggestion
3. Initialize the client and point it to the Miden testnet IP: `18.203.155.106`
```

      ```shell
      miden-client init
      ```

>    The command will set up the client. You can accept the default by pressing Enter. 

Reviewer: ```suggestion
   The command sets up the client. Accept the defaults by pressing _Enter_. 
```

!!! tip "The importance of syncing"
      - As you can see, the listed note is lacking a `commit-height`. 
      - This is because you have received a private note but have not yet synced your view of the rollup to check that the note is the result of a valid transaction.
      - Hence, before consuming the note we will need to update our view of the rollup by syncing.

>       - Many users could have gotten the same private note, we cannot know, but only one user will be able to consume the note in a transaction that gets verified by the Miden operator.

Reviewer: ```suggestion
      - Many users could have received the same private note, but only one user can consume the note in a transaction that gets verified by the Miden operator.
```

State synced to block 179672
```

> And now you note should have a `Commit Height`.

Reviewer: ```suggestion
And now your note should have a `Commit Height`.
```
